### DISCUSSION TITLE
Welcome to com.akylas.documentscanner Discussions!

### DISCUSSION THREAD

**Discussion by farfromrefug** [Akylas/OSS-DocumentScanner#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by djal1e** [comment]
Salut, elle est de plus en plus nickel ton application .
Pour l'instant je reste sur note bloc scanner (avec un bon anti pub), mais je commence doucement la bascule.
2 point pour moi qui me gène ,
_pas possible de créer des dossiers
_je ne gère pas le webdav, c'est pas encore ds mes cordes, du coup je peux pas faire de sauvegarde.

Merci pour l'App en tout cas,
Steve (Montpellier)

**Reply by farfromrefug** [reply]
hey I ll reply in English so that everyone can understand ;)
thanks happy to hear you like it 
about folders. I won't add folders per say but the code for "grouping" is already there . just need to wrap the UI. will try to tackle this soon!

about WebDAV / syncing what do you use right now ? 
I dont plan on adding google or Microsoft stuff as it is again what I stand for. but I want to make it easier for everyone. I am opened to suggestions . someone talked about rclone I need to look into this. if you have ideas please share
